templates/default/src/__tests__/pages/example/second-page/__snapshots__/index.test.jsx.snap

```diff
@@ -1,19 +1,27 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Pages:SecondPage renders a second example page 1`] = `
-<main>
-  <p>
-    This is another page at a different URL.
-  </p>
-  <p>
-    Return to the
-    <a
-      href="/"
-      onClick={[Function]}
-      onMouseEnter={[Function]}
-    >
-      homepage
-    </a>
-  </p>
-</main>
+<div
+  style={
```

Comment:
If we're using NextJS styled-jsx, then we shouldn't be getting inline CSS like this